socorro/signature/rules.py

```diff
@@ -569,13 +574,16 @@ def action(self, raw_crash, processed_crash, notes):
 
 
 class SigTrunc(Rule):
-    """ensure that the signature is never longer than 255 characters"""
+    """Truncates signatures down to SIGNATURE_MAX_LENGTH characters
```

Comment:
The original bug claims that signature length is and should be configurable, but it's not; it's an in-code constant set to 255. Is this intentional?

Also this claims to use `SIGNATURE_MAX_LENGTH` but uses a hard-coded length instead.

Reply:
It is intentional--that's something that changed when I redid signature generation over the summer and took out all the configuration. We've always had it as 255 and never changed it. We'd want any changes to be deployable and backed-outable. For something like this, seems better to set a constant in code.

However, it should use `SIGNATURE_MAX_LENGTH`--that's a mistake. I'll fix that.